components/Menu/Menu.js

```diff
@@ -99,14 +119,28 @@ export default function Menu({
               collections={collections}
               hasSubmenu={true}
               page="gaming"
-              menuMode="gaming"
               submenuMode={submenuMode}
               changeSubmenuMode={changeSubmenuMode}
               onToggleMenu={handleToggleMenu}
               changeFlashcardSelection={changeFlashcardSelection}
               getAllFlashcardsFromCollection={getAllFlashcardsFromCollection}
               closingTrigger={closingTrigger}
+              disabled={!session}
             />
+
+            <StyledMenuLogin
+              onClick={() => {
+                changeSubmenuMode("default");
+                changeFlashcardSelection("all");
+                handleToggleMenu();
+              }}
+            >
+              <Login
+                position="menu"
+                changeShowLogOutDialog={changeShowLogOutDialog}
+                showLogOutDialog={showLogOutDialog}
+              />
+            </StyledMenuLogin>
```

Comment:
The MenuLogin is an li but you added an onClick handler. This is not accessible. Please move the onClick logic to the Login button instead. You can pass an "onClick" prop into that Component and call it with the other internal logic inside the same onClick handler function.
```suggestion
            <StyledMenuLogin>
              <Login
                position="menu"
                changeShowLogOutDialog={changeShowLogOutDialog} // can be removed after moving state into Login
                showLogOutDialog={showLogOutDialog} // can be removed after moving state into Login
                onClick={() => {
                    changeSubmenuMode("default");
                    changeFlashcardSelection("all");
                    handleToggleMenu();
              }}
              />
            </StyledMenuLogin>
```